api/v1/swagger.yaml

```diff
@@ -1988,27 +1997,632 @@ paths:
+      operationId: deleteGeneratorWriter
+      parameters:
+        -
+          name: key
+          in: path
+          description: >-
+            The id or name of the generator
+          required: true
+          type: string
+        -
+          name: userNameOrId
+          in: path
+          description: >-
+            The username or the Id of the user
+          required: true
+          type: string
+      responses:
+        204:
+          description: >-
+            The user’s write permission has been revoked for the generator.
+        400:
+          $ref: "#/responses/400"
+        403:
+          $ref: "#/responses/403"
+        404:
+          $ref: "#/responses/404"
+        default:
+          $ref: "#/responses/genericError"
+
+  # ---------------------------------------------------------------------------
+
+  /generatorTemplates:
+    x-swagger-router-controller: generatorTemplates
+    get:
+      security:
+        - jwt: []
+      summary: Get a generatorTemplate or all generatorTemplates.
+      tags: [ generatorTemplates ]
+      description: >-
+        Get a generatorTemplate or all generatorTemplates. You may query using
+        field filters with asterisk (*) wildcards. You may also optionally
+        specify sort, limit, offset, and a list of fields to include in the
+        response. If the Refocus configuration parameter `useAccessToken` is
+        set to `true`, you must include an `Authorization` request header with
+        your [JSON Web Token](https://tools.ietf.org/html/rfc7519) (JWT) as the
+        value. You can get a token using `POST /v1/register` or `POST /v1/tokens`.
+      externalDocs:
+        description: >-
+          GET /generatorTemplates
+        url: http://focus.salesforce.com/docs/api/generatorTemplates
+      operationId: findGeneratorTemplates
+      parameters:
+        -
+          $ref: "#/parameters/GeneratorTemplateFieldsParam"
+        -
+          name: sort
+          in: query
+          description: >-
+            Specify the sort order using a field name, e.g. '...?sort=name'.
+            Prepend the field name with a minus sign to specify descending order,
+            e.g. '...?sort=-name'.
+          required: false
+          type: array
+          collectionFormat: csv
+          items:
+            type: string
+            enum:
+              - description
+              - name
+              - version
+              - createdAt
+              - updatedAt
+              - -description
+              - -name
+              - -version
+              - -createdAt
+              - -updatedAt
+        -
+          $ref: "#/parameters/limitParam"
+        -
+          $ref: "#/parameters/offsetParam"
+        -
+          name: description
```

Comment:
I don't think we are allowing query parameter filtering on description fields anywhere